packages/core/src/scenes/TimeEvents.ts

```diff
@@ -109,6 +112,18 @@ export class TimeEvents {
    * @internal
    */
   public register(name: string): number {
+    if (!this.collisionLookup.has(name)) {
+      this.collisionLookup.add(name);
+    } else {
+      useLogger().error(
+        'name: "' +
+          name +
+          '"' +
+          ' has already been used for another event name.',
+      );
```

Comment:
Three things:
- you can access the logger through `this.scene.project.logger`
- it's better to use template literals for messages like this:
- you can add a stack trace to the log to show people the where in the code this event was:
```suggestion
      this.scene.project.logger.error({
        message: `name "${name}" has already been used for another event name.`,
        stack: new Error().stack,
      });
```